import/party_scraper/src/party_scraper/pipelines.py

```diff
@@ -29,10 +29,16 @@
 class JsonWithEncodingPipeline(object):
     def _post_elastic(self, content):
 
+        # generate unique id for event
+        eventId = 'pa_scr_' + content['title'] + content['start_date']
```

Comment:
Geht das auch noch wenn der Titel Leerzeichen und/oder Umlaute enthält?

Reply:
Ja, das ist kein problem,habs ausprobiert und bei den anderen scrapern ist es genauso. 
Ist natürlich jetzt nicht so super hübsch, aber funktioniert.
Man könnte ein md5 drauf machen, ich würde das aber gern auf die nächste iterationsstufe schieben und das vielleicht vorher mal in größerer Runde diskutieren, wie wir mit den IDs weiter verfahren möchten.